src/PostgREST/Auth.hs

```diff
@@ -71,8 +70,8 @@ jwtClaims secret input time =
   value2map _          = M.empty
 
 {-| Receives the name of a role and returns a SET ROLE statement -}
-setRole :: Text -> BS.ByteString
-setRole r = "set local role " <> cs (pgFmtLit r) <> ";"
+setRole :: Value -> BS.ByteString
```

Comment:
Then again if `setRole` handles all claims including the role then we'll have to sort role to be the first one, so the code might be less understandable.